include/itkAttenuationImageFilter.hxx

```diff
@@ -126,8 +126,9 @@ AttenuationImageFilter<TInputImage, TOutputImage, TMaskImage>::BeforeThreadedGen
 
 template <typename TInputImage, typename TOutputImage, typename TMaskImage>
 void
-AttenuationImageFilter<TInputImage, TOutputImage, TMaskImage>::DynamicThreadedGenerateData(
-  const OutputRegionType & regionForThread)
+AttenuationImageFilter<TInputImage, TOutputImage, TMaskImage>::ThreadedGenerateData(
+  const OutputImageRegionType & regionForThread,
+  ThreadIdType                  threadId)
```

Comment:
As `threadId` parameter is not used, we should remove its name to avoid unused variable warning:
```suggestion
  ThreadIdType)
```

Reply:
Resolved in forced update.